### ISSUE TITLE
Color contrast logic does not always provide results that pass accessibility

### ISSUE THREAD

**Issue by nicolepaschen** [BoldGrid/post-and-page-builder#463]
When testing my sites to pass color contrast for accessibility I noticed the default provided by our system does not always pass - the default orange is a good example we have white text but it only passes with black text.
